orc8r/tools/packer/vagrant-box-upload.sh

```diff
@@ -21,33 +18,11 @@ if [ -z "$VAGRANT_CLOUD_TOKEN" ]; then
   exit 1
 fi
 
-echo -n "Creating the box for $USER/$BOX..."
-curl --header "Content-Type: application/json" --header "Authorization: Bearer $VAGRANT_CLOUD_TOKEN" https://app.vagrantup.com/api/v1/boxes --data '{ "box": { "username": "'$USER'", "name": "'$BOX'" } }'
-#{"errors":["Type has already been taken"],"success":false}
-
-echo -en "\nCreate the version $VERSION..."
-curl --header "Content-Type: application/json" --header "Authorization: Bearer $VAGRANT_CLOUD_TOKEN" https://app.vagrantup.com/api/v1/box/$USER/$BOX/versions --data '{ "version": { "version": "'$VERSION'" } }'
-#{"errors":["Version has already been taken"],"success":false}
-
-echo -en "\nCreating the $BOX_PROVIDER provider..."
-curl --header "Content-Type: application/json" --header "Authorization: Bearer $VAGRANT_CLOUD_TOKEN" https://app.vagrantup.com/api/v1/box/$USER/$BOX/version/$VERSION/providers --data "{ \"provider\": { \"name\": \"$BOX_PROVIDER\" }  }"
-#{"errors":["Metadata provider must be unique for version"],"success":false}
-
-echo -en "\nReceiving the upload url..."
-response=$(curl -s --header "Authorization: Bearer $VAGRANT_CLOUD_TOKEN" https://app.vagrantup.com/api/v1/box/$USER/$BOX/version/$VERSION/provider/$BOX_PROVIDER/upload)
-
-upload_path=$(echo "$response" | jq .upload_path | tr -d '"')
-
-echo -en "\nUploading ..."
-curl $upload_path -o upload.log --progress-bar --request PUT --upload-file $BOX_FILE
-
-# When the upload finishes, you can verify it worked by making this request and matching the hosted_token it returns to the previously retrieved upload token.
-uploaded_path=$(curl -s https://app.vagrantup.com/api/v1/box/$USER/$BOX/version/$VERSION/provider/$BOX_PROVIDER | jq .hosted_token | tr -d '"')
-
-if [[ $upload_path == $uploaded_path ]]; then
-    echo -en "\nReleasing..."
-    curl --header "Authorization: Bearer $VAGRANT_CLOUD_TOKEN" https://app.vagrantup.com/api/v1/box/$USER/$BOX/version/$VERSION/release --request PUT
-    echo "Done!"
-else
-    echo -e "\nUpload did not work: $upload_path != $uploaded_path"
-fi
+vagrant cloud auth login --token "$VAGRANT_CLOUD_TOKEN"
+vagrant cloud publish \
+    --release \
+    --force \
```

Comment:
I remember we discussed yesterday if `force` was really necessary. Can you remind me why we need it?

Reply:
I amended the commit to remove the `--force`